changelogs/unreleased/gh-7747-seqscan.md

```diff
@@ -0,0 +1,4 @@
+## feature/sql
+
+* A new SEQSCAN keyword has been introduced and a new restrictions on scanning
+  SELECTs has been applied (gh-7747).
```

Comment:
Shouldn't it be marked as a breaking change?

Reply:
Added.